src/ninshiki_cpp/detector/dnn_detector.cpp

```diff
@@ -46,7 +46,16 @@ DnnDetector::DnnDetector()
     classes.push_back(line);
   }
 
-
+  std::ifstream ifs_cfg(config.c_str());
+  std::string cfg_line;
+  while (std::getline(ifs_cfg, cfg_line)) {
+    if (cfg_line.find("width") != std::string::npos) {
+      width = std::stoi(jitsuyo::split_string(cfg_line, "="));
+    } else if (cfg_line.find("height") != std::string::npos) {
+      height = std::stoi(jitsuyo::split_string(cfg_line, "="));
+      break;
+    }
+  }
```

Comment:
This loop assumes that ```width``` is always above ```height``` in the cfg. You can do something like this in case the cfg is written otherwise:
```suggestion
 int flag = 0;
  while (std::getline(ifs_cfg, cfg_line)) {
    if (flag >= 2) break;
    if (cfg_line.find("width") != std::string::npos) {
      width = std::stoi(jitsuyo::split_string(cfg_line, "="));
      flag++;
    } else if (cfg_line.find("height") != std::string::npos) {
      height = std::stoi(jitsuyo::split_string(cfg_line, "="));
      flag++;
    }
  }
```

Reply:
```cpp
if (height != 0 && width != 0) {
  break;
}
```
instead of `flag`, what about this?

Reply:
ow yeah, that's better